dora/stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java

```diff
@@ -319,21 +412,17 @@ private void runInternal() throws Exception {
       }
       CommonUtils.sleepMs(waitMs);
 
-      int i = 0;
       while (!Thread.currentThread().isInterrupted()
-          && CommonUtils.getCurrentMs() < mContext.getEndMs() && i < mFilePaths.length) {
```

Comment:
Before this change, technically the BenchThread keeps reading a bunch of files in a round-robin fashion. After this change, each thread keeps reading the same file again and again. 

I can actually change it so that a BenchThread reads multiple files, but is that necessary?